static/module.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<!--To test ECMA modules, run `npm run build` and then `npm start`
+    in the root KaTeX directory and then visit with a web browser
+    which supports modules: http://localhost:7936/module.html
+  -->
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>KaTeX Test</title>
+        <link rel="stylesheet" type="text/css" href="/build/katex.css">
+        <link rel="stylesheet" type="text/css" href="/main.css">
+        <script type="module" type="text/javascript">
+            import katex from '/build/katex.mjs';
```

Comment:
How is this file being built, I thought `webpack` didn't output ES6 modules?

Reply:
@kevinbarabash The files are built using `babel` directly with plugins to change `import` paths and rename files.

If we're to distribute the bundled file, we can use `rollup` and simplify the process.

Reply:
I think distributing a single bundle for the ES module is probably the way to go.  That way people can use it in production.

Reply:
@kevinbarabash Switched to rollup to bundle the ES module. I'm surprised how light and fast rollup is 😄